c2corg_ui/static/js/utils.js

```diff
@@ -123,6 +123,26 @@ app.utils.formatDate = function(date) {
 
 
 /**
+ * check that the start date if before end date.
+ * Return true is start > end
+ * @export
+ * @param {Object} outing
+ * @return {(boolean | undefined)}
```

Comment:
Why is the type `boolean | undefined`? Isn't `boolean` enough? You just have to make sure that your function always returns a boolean value.